src/main/java/org/octri/hpoonfhir/controller/LaunchController.java

```diff
@@ -38,14 +49,21 @@
 	@GetMapping("/launch")
 	public void launch(HttpServletRequest request, HttpServletResponse response) {
 
-		String launch = request.getParameter("launch");
-		String serviceUri = request.getParameter("iss");
+        String launch = request.getParameter(LAUNCH_ID_PARAMETER);
```

Comment:
It looks like there may be some whitespace issues with this line. You might want to run the formatter.